src/runtime_src/core/tools/xbutil2/SubCmdValidate.cpp

```diff
@@ -842,8 +842,15 @@ dmaTest(const std::shared_ptr<xrt_core::device>& _dev, boost::property_tree::ptr
       break;
   }
 
+  auto is_host_mem = [](std::string tag) {
+    return tag.compare(0,4,"HOST") == 0;
```

Comment:
This line pretty much doesn't do anything for this function doe not return a value.

Reply:
Opps...my bad...this is a LAMDA.